src/scripts/main.js

```diff
@@ -1,3 +1,350 @@
 'use strict';
 
-// write your code here
+const start = document.querySelector('.start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+const score = document.querySelector('.game-score');
+let allRows = document.querySelectorAll(`.field-row`);
+let countScore = 0;
+let wasMove = false;
+let restartButton;
+
+start.addEventListener('click', () => {
+  reset();
+  randomCell();
+  randomCell(true);
+  document.addEventListener('keydown', listener);
+});
+
+function listener(e) {
+  e.preventDefault();
+
+  const allCells = document.querySelectorAll(`.field-cell`);
+
+  allRows = document.querySelectorAll(`.field-row`);
+
+  const activeCells = [...allCells].filter(elem => {
+    return elem.innerHTML !== '';
+  });
+
+  if (e.code === 'ArrowLeft') {
```

Comment:
Move all directions to the object.

Example of usage:
`e.code === Directions.ArrowLeft`